hw2/src/main/java/org/example/Main.java

```diff
@@ -80,6 +80,21 @@ public static void main(String[] args) {
         System.out.println("Разница мужду массами двух бойцов: "  + (secondBoxer - firstBoxer) + " кг");
         System.out.println("Второй способ решения задачи №7.");
         System.out.println("Разница мужду массами двух бойцов: "  + (secondBoxer%firstBoxer) + " кг");
+        System.out.println();
+        /*   Задача 8.
+    Решите задачу с помощью функций сложения, вычитания, умножения и деления.
 
+            640 часов работы поделено между сотрудниками.
+            Если каждый сотрудник посвящает работе 8 часов, то сколько всего работников в компании?
+            Выведите результат задачи в консоль в формате: «Всего работников в компании — … человек».
+    Посчитайте, сколько часов работы должно быть поделено между сотрудниками, если в компании работает на 94 человека больше.
+    Выведите результат задачи в консоль в формате: «Если в компании работает … человек,
+    то всего … часов работы может быть поделено между сотрудниками».
+ */
+        var totalHours = 640;
+        var hours = 8;
+        System.out.println("Всего работников в компании - " + totalHours/hours + " человек");
+        var emploeers = totalHours/hours + 94;
+        System.out.println("Если в компании работает - " + emploeers + "  человек, то всего - " + emploeers*8 + " часов работы может быть поделено между сотрудниками.");
```

Comment:
снова не используется переменная
```suggestion
        System.out.println("Если в компании работает - " + emploeers + "  человек, то всего - " + emploeers * hours + " часов работы может быть поделено между сотрудниками.");
```